detect_secrets/audit/io.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, allow_labelling: bool, allow_backstep: bool) -> None:
 
     def __str__(self) -> str:
         if 'Y' in self.valid_input:
-            output = 'Is this a secret that should be committed to this repository?'
+            output = 'Should this be committed to the repository?'
```

Comment:
Could we change it to the following (in all occurrences)?
```
Should this string be committed to the repository?
```

Reply:
Sure, that's fine by me. I'll update the branch.